library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -192,6 +191,82 @@ libspdm_build_opaque_data_supported_version_data(libspdm_context_t *spdm_context
     return LIBSPDM_STATUS_SUCCESS;
 }
 
+
+/**
+ * Get element from multi element opaque data by element id.
+ *
+ * This function should be called in
+ * libspdm_process_opaque_data_supported_version_data/libspdm_process_opaque_data_version_selection_data.
+ *
+ * @param[in]  data_in_size                size of multi element.
+ * @param[in]  data_in                     A pointer to the multi element.
+ * @param[in]  element_num                 element number.
+ * @param[in]  element_id                  element id.
+ * @param[out] get_element_ptr             pointer to store finded element
+ *
+ * @retval true                            get element successfully
+ * @retval false                           get element failed
+ **/
+bool libspdm_get_element_from_opaque_data(size_t data_in_size, const void *data_in,
+                                          uint8_t element_num, uint8_t element_id,
```

Comment:
I dont think we need element_num here. Please remove it.

Element_num should be get from the data_in.